docs/v8-migration-api.md

```diff
@@ -942,6 +942,16 @@ Most of the changes to our TypeScript-specific AST nodes are to reduce the diffe
 
   __Migration__: Call `t.identifier` with a valid name.
 
+- Reject invalid variable declarator in `t.variableDeclaration` builder ([#10917](https://github.com/babel/babel/pull/10917), [#17217](https://github.com/babel/babel/pull/17217))
+
+  ```js title="babel-plugin.js"
+  // var [x] is invalid without an initializer
+  // highlight-error-next-line
+  t.variableDeclaration("var", [
+    t.variableDeclarator(t.arrayPattern([t.identifier("x")]))
+  ]);
```

Comment:
Maybe highlight these three lines?

Reply:
Currently we don't have highlight-error block. I will see if I can add it.

Reply:
We can also copy the comment three times. :)